enh(release): prepare release notes for engine 20.04.12 (#780)
centreon/centreon-documentation: en/releases/centreon-core.md

## Centreon Engine

### 20.04.12

#### Bugfixes

> - Engine burns 100% of the CPU when check_period set to none

Reviewer: ```suggestion
- Engine cpu usage increased to 100% when `check_period` is set to `none`
```

## Centreon Engine

### 20.04.12

> > This version requires Centreon Clib and Centreon-Connector versions to be 20.04.2 or higher.

Reviewer 2: ```suggestion
> This version requires Centreon Clib version to be 20.04.2 or higher.
> This version requires Centreon-Connector version to be 20.04.1 or higher.
```

## Centreon Engine

### 20.04.12

> 

Reviewer: ```suggestion

`June 4, 2021`

```
